sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -2197,6 +2390,23 @@ public partial class ExtensionDataSource : System.ClientModel.Primitives.IJsonMo
         public static bool operator !=(Azure.ResourceManager.Monitor.Models.ExtensionDataSourceStream left, Azure.ResourceManager.Monitor.Models.ExtensionDataSourceStream right) { throw null; }
         public override string ToString() { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ExternalNetworkingMode : System.IEquatable<Azure.ResourceManager.Monitor.Models.ExternalNetworkingMode>
```

Comment:
Could we specify the name of this type?

Reply:
Renamed to `PipelineGroupExternalNetworkingMode`